examples/zig/src/main.zig

```diff
@@ -0,0 +1,10 @@
+const std = @import("std");
+
+pub fn main() anyerror!void {
+    const stdout = std.io.getStdOut().writer();
+    nosuspend stdout.print("All your codebase are belong to us.\n", .{}) catch return;
```

Comment:
Change what is printed to be consistent with the rest of the examples.

```suggestion
    nosuspend stdout.print("Hello from Zig\n", .{}) catch return;
```